examples/render_grid/run.py

```diff
@@ -0,0 +1,334 @@
+# Before running this file, rename user_config.yaml.example -> user_config.yaml and modify it with appropriate paths for your system.
+
+# Before running this example, you should have a good understanding of the example in render_image/run.py
+
+import math
+import os
+
+import cv2
+import numpy as np
+
+import spear
+
+if __name__ == "__main__":
+
+    # define the number of cameras in the grid
+    rows = 3
+    cols = 3
+
+    # create instance
+    config = spear.get_config(
+        user_config_files=[
+            os.path.realpath(
+                os.path.join(os.path.dirname(__file__), "user_config.yaml")
+            )
+        ]
+    )
+    spear.configure_system(config=config)
+    instance = spear.Instance(config=config)
+
+    # initialize actors and components
+    with instance.begin_frame():
+        # find functions
+        AActor = instance.unreal_service.get_static_class(class_name="AActor")
+        AActor_K2_SetActorLocation = instance.unreal_service.find_function_by_name(
+            uclass=AActor, function_name="K2_SetActorLocation"
+        )
+        AActor_K2_SetActorRotation = instance.unreal_service.find_function_by_name(
+            uclass=AActor, function_name="K2_SetActorRotation"
+        )
+
+        UGameplayStatics = instance.unreal_service.get_static_class(
+            class_name="UGameplayStatics"
+        )
+        UGameplayStatics_GetPlayerController = (
+            instance.unreal_service.find_function_by_name(
+                uclass=UGameplayStatics, function_name="GetPlayerController"
+            )
+        )
+
+        USpSceneCaptureComponent2D = instance.unreal_service.get_static_class(
+            class_name="USpSceneCaptureComponent2D"
+        )
+
+        USpSceneCaptureComponent2D_Initialize = (
```

Comment:
Consistent naming with other examples (including some that aren't in `main` yet):

```
sp_scene_capture_component_2d_initialize_func = ...
sp_scene_capture_component_2d_terminate_func = ...
```